tensorflow/python/lib/io/file_io.py

```diff
@@ -336,24 +340,29 @@ def write_string_to_file(filename, file_content):
 
 
 @tf_export(v1=["gfile.Glob"])
-def get_matching_files(filename):
+def get_matching_files(filename, max_parallel_num=1):
```

Comment:
This should be `num_threads` (the max mention is self-evident and thus not necessary)

Note: `num_threads` / `num_workers` is already an argument in several places.

Reply:
All right! Done.